covid/transform.py

```diff
@@ -858,3 +920,17 @@ def indication_of_rebound(series_):
             indicator = "Rebound"
 
     return indicator
+
+
+def calculate_cdc_criteria_positive_and_negative_streak_series(criteria_series):
```

Comment:
nit: should we move this to `transform_utils` and add a quick unit test? both are unnecessary, so don't worry about it if that seems over the top!

Reply:
Good idea! On it!